zuul.yaml

```diff
@@ -35,6 +35,44 @@
         comment: true
         status_url: *log_link
 
+- pipeline:
+    name: gate
+    manager: dependent
+    allow-secrets: True
+    trigger:
+      github:
+        - event: pull_request
+          action: status
+          status: "bonnyci[bot]:check:success"
+        - event: pull_request
+          action: comment
+          comment: (?i)^\s*gate-recheck\s*$
+        - event: pull_request_review
+          action: submitted
+          state: approved
+        - event: pull_request_review
+          action: dismissed
+          state: request_changes
+    require:
+      status: "bonnyci[bot]:check:success"
+      approval:
+        - review: 2
+    start:
+      github:
+        status: pending
+        comment: false
+    success:
+      github:
+        status: success
+        comment: false
+        status_url: *log_link
+        merge: true
```

Comment:
for now what do you think about leaving merge: false? This would let us do some testing without having to create actual new patches each time.